specificatie/openapi.yaml

```diff
@@ -99,7 +97,7 @@ paths:
           required: false
           schema:
             type: string
-            maxLength: 40
+            pattern: ^(\?+|\*)?[a-zA-Z0-9À-ÿ' '\-\']{1,40}(\?+|\*)?$
```

Comment:
niet gelijk aan de pattern in de request body. De ' voor en achter de spatie kan weg.

Reply:
Aangepast